DEPS

```diff
@@ -127,7 +127,7 @@ deps = {
    Var('llvm_git') + '/llvm-project/libcxx' + '@' + '44079a4cc04cdeffb9cfe8067bfb3c276fb2bab0',
 
   'src/third_party/libcxxabi':
-   Var('flutter_git') + '/third_party/libcxxabi' + '@' + '483f071ff4780a8884f32d97d2d262fbe9f1ae18',
+   Var('llvm_git') + '/llvm-project/libcxxabi' + '@' + '2ce528fb5e0f92e57c97ec3ff53b75359d33af12',
```

Comment:
Hm, good question. `llvm_git` was introduced in https://github.com/flutter/engine/pull/34720


@godofredoc any idea why this works when `llvm.googlesource.com` isn't in `allowed_hosts`?
https://github.com/flutter/engine/blob/1eeb38cc99a847c0a5030d3587c48b1d156b4f04/DEPS#L18
https://github.com/flutter/engine/blob/1eeb38cc99a847c0a5030d3587c48b1d156b4f04/DEPS#L103-L109

I believe this logic is in https://chromium.googlesource.com/chromium/tools/depot_tools.git/+/refs/heads/main/gclient.py#931

Reply:
It looks like this is only validated during `gclient verify`, and it seems we aren't running that in CI.  The infra team has been informed about this and I filed https://github.com/flutter/flutter/issues/113052.

In the meantime this PR is no worse than what's already on master since `libcxx` is also using `llvm_git`, so I don't think we should block this PR on fixing that.
https://github.com/flutter/engine/blob/966f249fb132df0f91d326c022adaa0f8d52dd77/DEPS#L126-L127